### PR TITLE
06-TimeManagement-ModuleAbsoluteTime: add implementation

### PR DESCRIPTION
**Pull request by prosolvo4th**

### CODE REVIEW

**Comment by prosolvo4th** [comment]
Changes: fix errors generated by checkpatch.pl

**Comment by prosolvo4th** [comment]
Changes: change printing format of ns from '%ld' to '%06ld', due to ignoring high zero

File: 06-TimeManagement-ModuleAbsoluteTime/absolute-time.c [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+struct timespec ts;
+
+
+static ssize_t ro_show(struct class *class,
+		struct class_attribute *attr,
+		char *buf)
+{
+	char text[100];
+
+	sprintf(text, "%ld.%06ld\n", ts.tv_sec, ts.tv_nsec);
```
Comment (yekovalyov) [inline 06-TimeManagement-ModuleAbsoluteTime/absolute-time.c]: > sprintf(text, "%ld.%06ld\n", ts.tv_sec, ts.tv_nsec);
Is this format string correct for values in ts structure?

Comment (prosolvo4th) [reply 06-TimeManagement-ModuleAbsoluteTime/absolute-time.c]: yes. it is.

Comment (prosolvo4th) [reply 06-TimeManagement-ModuleAbsoluteTime/absolute-time.c]:
```
/home/the4th/Projects/GL/myFork/gl-kernel-training-2019.git/06-TimeManagement-ModuleAbsoluteTime/absolute-time.c:18:18: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘__kernel_time_t {aka long int}’ [-Wformat=]
  sprintf(text, "%d.%06ld\n", ts.tv_sec, ts.tv_nsec);
                 ~^           ~~~~~~~~~
                 %ld
```

Comment (prosolvo4th) [reply 06-TimeManagement-ModuleAbsoluteTime/absolute-time.c]: yes. It is.

Comment (prosolvo4th) [reply 06-TimeManagement-ModuleAbsoluteTime/absolute-time.c]:
```
/home/the4th/Projects/GL/myFork/gl-kernel-training-2019.git/06-TimeManagement-ModuleAbsoluteTime/absolute-time.c:18:18: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘__kernel_time_t {aka long int}’ [-Wformat=]
  sprintf(text, "%d.%06ld\n", ts.tv_sec, ts.tv_nsec);
                 ~^           ~~~~~~~~~
                 %ld
```

Comment (yekovalyov) [reply 06-TimeManagement-ModuleAbsoluteTime/absolute-time.c]: tv_sec is ok. What about %06ld  and tv_nsec?

Comment (prosolvo4th) [reply 06-TimeManagement-ModuleAbsoluteTime/absolute-time.c]: ok. I get it)

**Comment by prosolvo4th** [comment]
Changes: %06ld -> %09ld for printing ts.nses

**Pull request merged**